### DISCUSSION TITLE
UDP connection

### DISCUSSION THREAD

**Discussion by MyliumFPGA** [Serial-Studio/Serial-Studio#65] [Q&A]
Hi,
I'm using ESP32 to broadcast UDP data to 192.168.0.255:2000 and then capture data using Serial Studio (Network, socket type: UDP, IP address: 192.168.0.255, port 2000). Data seem to be broadcasted properly (I can see them using Wireshark, proper addresses -  destination IP: 192.168.0.255, proper data) but Serial Studio doesn't show anything. Am I missing something or doing it wrong?

Dominik

**Comment by bigjohnson** [comment]
See this [Issue](https://github.com/Serial-Studio/Serial-Studio/issues/74).

Alberto

**Comment by alex-spataru** [comment]
Hi, I totally screwed up by treating the UDP socket as a normal I/O device (like the TCP socket or the serial port). I am currently working on adding full support for UDP connections. See my [comment](https://github.com/Serial-Studio/Serial-Studio/issues/74#issuecomment-938334060) on issue #74 for more information.
